apps/browser/qml/pages/BrowserPage.qml

```diff
@@ -139,6 +139,9 @@ Page {
             PropertyChanges {
                 target: webView.contentItem
                 virtualKeyboardMargin: virtualKeyboardObserver.panelSize
+                virtualKeyboardHeight: virtualKeyboardObserver.opened
+                                       ? virtualKeyboardObserver.imSize
+                                       : 0
```

Comment:
```
virtualKeyboardHeight: virtualKeyboardObserver.panelSize
```
See when condition. Further, given that virtualKeyboardHeight works is it should you should remove virtualKeyboardMargin (setting margin as well) + cleanup related sendVkbOpenCompositionMetrics from browser side and "embedui:vkbOpenCompositionMetrics" handling from the embedlite-components